src/pybind/mgr/cephadm/utils.py

```diff
@@ -97,3 +98,7 @@ def str_to_datetime(input: str) -> datetime.datetime:
 
 def datetime_to_str(dt: datetime.datetime) -> str:
     return dt.strftime(DATEFMT)
+
+
+def resolve_ip(hostname: str) -> str:
+    return socket.getaddrinfo(hostname, 2049, flags=socket.AI_CANONNAME, type=socket.SOCK_STREAM)[0][4][0]
```

Comment:
this is duplicated. we know all IPs for a host already:

https://github.com/ceph/ceph/blob/ba99f56f52c01c7bea1ddd677746150ea1265ee0/src/pybind/mgr/cephadm/inventory.py#L174